addons/spreadsheet/static/src/global_filters/components/filter_date_from_to_value/filter_date_from_to_value.js

```diff
@@ -0,0 +1,44 @@
+/** @odoo-module */
+
+import { Component, onWillUpdateProps } from "@odoo/owl";
+import { DateTimeInput } from "@web/core/datetime/datetime_input";
+import { serializeDate, deserializeDate } from "@web/core/l10n/dates";
+
+export class DateFromToValue extends Component {
+    setup() {
+        this._setStateFromProps(this.props);
+        onWillUpdateProps(this._setStateFromProps);
```

Comment:
we don't need the state and the `onWillUpdateProps`. We can always rely on the props.
Instead of setting the state, then call `onFromToChanged` with the values from the state, just call `_updateFilter` with the new values